### ISSUE TITLE
Accessibility review

### ISSUE THREAD

**Issue by kerchner** [gwu-libraries/sfm-ui#1004]
- Use Siteimprove browser widget for Chrome
- Do this after #987 
- This ticket is for review; remediations will be a separate issue(s)

Also see issues identified in initial review:  https://gwlai.monday.com/boards/400282245

**Comment by kerchner** [comment]
Do this after #987

**Comment by lwrubel** [comment]
Initial accessibility review using SiteImprove extension:
https://docs.google.com/document/d/187sklzhve5K5lkks77DWHKu3fZgRMzA1Y49eHjqSrAI/

**Comment by lwrubel** [comment]
This ticket was for the review of accessibility issues, which is completed and documented in the Google Doc and Monday boards linked in the thread. Accessibility issues were addressed by #1022, #1023, #1024, #1025, #1026.
